pkg/provider/provider.go

```diff
@@ -93,7 +95,7 @@ func NewVirtualK8S(cfg provider.InitConfig, cc *ClientConfig,
 		return nil, fmt.Errorf("could not get target cluster server version: %v", err)
 	}
 
-	informer := kubeinformers.NewSharedInformerFactory(client, 0)
+	informer := kubeinformers.NewSharedInformerFactory(client, cc.ClientInformerResyncPeriod)
```

Comment:
Based on current implementation, we do not need resync. It is just for computing node resource. Also, the idea scene is that we only create/delete pods from upper clusters.